.cmake/target.cmake

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
```

Comment:
On Linux the processor is reported as "aarch64". Better match both "arm64" and "aarch64".